tests/std/expected_results.txt

```diff
@@ -1,2 +1,13 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# TRANSITION, VSO-1093670 "Memory corruption in EDG when processing multiple TUs"
+tests/GH_000545_include_compare:15 SKIP
+tests/GH_000545_include_compare:16 SKIP
+tests/P0607R0_inline_variables:19 SKIP
+tests/P0607R0_inline_variables:20 SKIP
+tests/P0607R0_inline_variables:21 SKIP
+
+# TRANSITION, GH-697 "We should have means of collecting compiler and test crash dumps"
+# The ICE we are skipping this test because of is not easily reproducible we need to collect dumps

```

Comment:
Grammar issues, but as this isn't product code, I won't ask you to reset testing for this.

Reply:
<3